vdr/store/store.go

```diff
@@ -288,6 +294,10 @@ func (s *store) Resolve(id did.DID, metadata *vdr.ResolveMetadata) (returnDocume
 				return err
 			}
 
+			if metadataRecord.Deactivated && metadata == nil {
```

Comment:
this belongs in a different PR

Reply:
Not really, since the in memory store did it correctly. With the change to the other store, tests fail.